prow/plugins/updateconfig/updateconfig.go

```diff
@@ -65,55 +66,62 @@ type kubeClient interface {
 }
 
 func handlePullRequest(pc plugins.PluginClient, pre github.PullRequestEvent) error {
-	configFile := pc.PluginConfig.ConfigUpdater.ConfigFile
-	pluginFile := pc.PluginConfig.ConfigUpdater.PluginFile
-	return handle(pc.GitHubClient, pc.KubeClient, pc.Logger, pre, configFile, pluginFile)
+	return handle(pc.GitHubClient, pc.KubeClient, pc.Logger, pre, maps(pc))
 }
 
-func handleConfig(gc githubClient, kc kubeClient, org, repo, commit, configFile string) error {
-	content, err := gc.GetFile(org, repo, configFile, commit)
-	if err != nil {
-		return err
+func maps(pc plugins.PluginClient) map[string]string {
+	log := pc.Logger
+	cu := pc.PluginConfig.ConfigUpdater
+	m := map[string]string{}
+	if cu.ConfigFile != "" {
+		log.Warnf(`config_file is deprecated, please switch to "maps": {"%s": "config"} before July 2018`, cu.ConfigFile)
```

Comment:
Lets update the Prow README too with a release note?

Reply:
Done